### PR TITLE
Update Parascopy to v1.9.1

### PR DESCRIPTION
**Pull request by tprodanov**
Update [Parascopy](https://github.com/tprodanov/parascopy) to version 1.9.0.

### CODE REVIEW

**Comment by dpryan79** [comment]
I haven't yet merged the htslib migration in, so you might wait for that, or instead manually pin to 1.16 in the recipe.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

File: recipes/parascopy/meta.yaml [hunk 1]
```diff
@@ -1,51 +1,62 @@
...
 
-source:
-  url: https://github.com/tprodanov/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 2161b28ea98a58cb283f5d452199ea29f15ff12cd372b333fc8b05072d3e2949
-
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --ignore-install --no-deps -vv 
+  skip: true # Temporarily skip osx
```
Comment (dpryan79) [inline recipes/parascopy/meta.yaml:10]:
```suggestion
  skip: true # [osx]
```

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
Will this break with htslib 1.16?

**Comment by tprodanov** [comment]
@dpryan79 Thank you for the answers! For now, I am okay with just publishing the linux version, and I will check if it is possible to create an osx version after a month.

**Comment by tprodanov** [comment]
@BiocondaBot please review & merge

**Comment by tprodanov** [comment]
@dpryan79 I already tried that, it fails.

**Comment by dpryan79** [comment]
Ah, it turns out that tabixpp is missing a build against htslib 1.15, which is what's used project-wide at the moment. I'll fix that.

**Comment by dpryan79** [comment]
The tabixpp fix seems to have solved the issue.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**